### PR TITLE
Update MockWebServer README with gradle and fix code

### PR DESCRIPTION
**Pull request by HugoGresse**
Add gradle as the recommanded download mode and remove unnecessary ")" for dispatcher example

### CODE REVIEW

File: mockwebserver/README.md [hunk 1]
```diff
@@ -140,8 +140,12 @@ server.setDispatcher(dispatcher);
 
 ### Download
 
-The best way to get MockWebServer is via Maven:
+The best way to get MockWebServer is via Gradle:
```
Comment (swankjesse) [inline mockwebserver/README.md]: Mind flipping these two? I don't want to imply endorsement of Gradle!

**Comment by HugoGresse** [comment]
I've updated the README according comment from @swankjesse and changed start() to play()

**Comment by swankjesse** [comment]
Awesome. Last request: can you please http://rebaseandsqua.sh this change?

**Comment by swankjesse** [comment]
I ran rebase & squash & committed this change.

Thank you.

**Comment by HugoGresse** [comment]
sorry I was in vacation for a week. Thank you for the merge.

**Pull request closed without merging**